Samples/kotlin/src/main/res/layout/activity_routing.xml

```diff
@@ -1,28 +1,63 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/content"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <fragment xmlns:android="http://schemas.android.com/apk/res/android"
+    <fragment
         android:id="@+id/map"
-        android:name="com.phunware.mapping.MapFragment"
+        android:name="com.phunware.mapping.SupportMapFragment"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
```

Comment:
is match_parent advised when all 4 edges are constrained to parent? I thought we were to use 0dp or something.

Reply:
Great catch.  Will fix.